src/components/ProposalImagePreview/ProposalImagePreview.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import Title from 'decentraland-gatsby/dist/components/Text/Title'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+
+import { ProposalAttributes } from '../../entities/Proposal/types'
+import ImageGallery from '../ImageGallery/ImageGallery'
+
+interface Props {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  proposal: ProposalAttributes<any>
+  isVisible: boolean
+}
+
+function ProposalImagePreview({ proposal, isVisible }: Props) {
+  const t = useFormatMessage()
+  return (
+    <>
+      {isVisible && !!proposal.configuration.image_previews && (
```

Comment:
I'd move this condition to where ProposalImagePreview is used. And avoid rendering this component if this condition is not met. And you can avoid passing isVisible as a prop.